drivers/sensor/bmi270/bmi270.c

```diff
@@ -545,13 +578,13 @@ static int bmi270_attr_set(const struct device *dev, enum sensor_channel chan,
 	    || (chan == SENSOR_CHAN_ACCEL_XYZ)) {
 		switch (attr) {
 		case SENSOR_ATTR_SAMPLING_FREQUENCY:
-			ret = set_accel_odr_osr(dev, val, NULL);
+			ret = set_accel_odr_osr(val, NULL, dev);
```

Comment:
I don't think this is a good change.

Having the device pointer first is a common C idiom for emulating object oriented programming. Placing it first is a signal that `dev` is the receiver.

Similarly for the other analogous changes.

Reply:
Ok, I will flip 'dev' to beginning in next push

Reply:
This still needs to be done